ocs_ci/ocs/resources/namespacestore.py

```
@@ -180,6 +178,19 @@ def verify_health(self, timeout=360, interval=5):
         except TimeoutExpiredError:
+            try:
+                nss_data = OCP(
+                    kind="namespacestore",
+                    namespace=config.ENV_DATA["cluster_namespace"],
+                    resource_name=self.name,
+                ).get()
+                phase = nss_data.get("status", {}).get("phase", "N/A")
+                conditions = nss_data.get("status", {}).get("conditions", [])
+                log.error(
+                    f"{self.name} stuck in phase '{phase}', conditions: {conditions}"
+                )
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)